packages/babel-core/src/index.js

```diff
@@ -22,6 +22,7 @@ export {
   loadOptions,
   loadOptionsSync,
   loadOptionsAsync,
+  getExternalDependencies,
```

Comment:
It is okay to export from a nested sub folder like `./config/helpers/config-api.js`.

Reply:
Fixed.